test/Combinators/testCombinator.ts

```diff
@@ -0,0 +1,57 @@
+import {ReasonGuard, constantGuards, unnegatableConstantGuards} from '../../src';
+import {assertGuards} from '../assertGuards';
+
+export type Tautology = ReasonGuard<unknown, unknown>;
+export const abbrev = (b: boolean) => b ? 'T' : 'F';
+
+export function testCombinator1(
```

Comment:
These should have more meaningful names

Reply:
Suggestions? `testOneArgCombinator` and `testTwoArgCombinator` seem more wordy without much more meaning, to me.

Reply:
Actually I think they're much better ... `testFoo1` and `testFoo2` at first glance just look like `first test of foo` and `second test of foo`, but the wordy names make clear that it's about what kind of `Foo` you're testing